module.nix

```diff
@@ -187,7 +187,7 @@ in {
       (mkIf cfg.postgres.enable_server {
         services.postgresql = {
           enable = true;
-          package = pkgs.postgresql_16;
+          package = pkgs.postgresql_17;
```

Comment:
I think postgresql isn't available in nixos yet, not even unstable has it. https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=postgresql_17

Also see #4227

Reply:
Yes, was bumping first just to see if the pipeline runs through, which is not the case...

Reply:
changed back to PG 16 for now.